server.js

```diff
@@ -26,34 +26,46 @@ var manifest = {
             }
         }
     ],
-
-    plugins: {
-
-        // General porpoise
-        './dogwater':   Config.dogwater,
-        './poop':       Config.poop,
-
-        // Server-specific
-        '../lib': [{ select: 'boilerplate-api' }]
-
-    }
+    registrations: [
+        {
+            plugin: {
+                register: './dogwater',
+                options: Config.dogwater
+            }
+        },
+        {
+            plugin: {
+                register: './poop',
+                options: Config.poop
+            }
+        },
+        {
+            plugin: {
+                register: '../lib',
+                options: [{ select: 'boilerplate-api' }]
+            }
+        }
+    ]
 
 };
 
 module.exports = manifest;
 
 // If this is being required, return the manifest.  Otherwise, start the server.
 if (!module.parent) {
-    Glue.compose(manifest, { relativeTo: Path.join(__dirname, 'node_modules') }, function (err, server) {
 
-        if (err) {
+    Glue.compose(manifest, { relativeTo: Path.join(__dirname, 'node_modules') }, (err, server) => {
 
+        if (err) {
             throw err;
         }
+        server.start( (err) => {
 
-        server.start(function () {
-
-            console.log('Started on ' + Config.server.boilerplateApi.uri);
+            if (err) {
+                console.log(err);
```

Comment:
Good catch... jeeeeeeeeeepers.  Can we just throw the error instead and let it not get caught?